code/obj/item/toilets.dm

```diff
@@ -117,7 +117,10 @@ TOILET
 				A.set_loc(target)
 #endif
 		src.clogged = 0
-		src.contents.len = 0
+		for (var/item in get_all_contents())
+			if (hud && item)
+				qdel(item)
+				src.hud.remove_item(item)
```

Comment:
```suggestion
		qdel(item)
		src.hud?.remove_item(item)
```
When iterating through `contents` item will never be null and I'd rather still delete the item if the hud is nonexistent (and remove from the hud conditionally).